src/Test/L0/Plugin/TestResultParser/PipelineTestRunPublisherL0.cs

```diff
@@ -76,7 +82,13 @@ public async Task PipelineTestRunPublisher_PublishTestRun_ForBatchedResults()
             var pipelineConfig = new PipelineConfig()
             {
                 BuildId = 1,
-                Project = new Guid()
+                Project = new Guid(),
+                StageName = "Stage1",
+                StageAttempt = 1,
+                PhaseName = "Phase1",
+                PhaseAttempt = 1,
```

Comment:
maybe create a default pipelineconfig object and in initialize or constructor and in the test override the props that are different from the defaults